### DISCUSSION TITLE
Magento 2.4.1: Class Dompdf/Dompdf does not exist

### DISCUSSION THREAD

**Discussion by sandeep570** [dompdf/dompdf#2634] [General]
Hello Guys,

It's really urgent that I need guidance from you guys is that I am using Magento 2.4.1 and I have installed the Dompdf using composer but still the class not found issue exists so could you please help me out on this?

Thanks in advance

**Comment by bsweeney** [comment]
Can't help much with specific Magento setup, but might be able to give you some pointers to help you figure out your issue. Which version of Dompdf are you referencing in our composer.json? And, are you referencing the composer autoloder?

**Reply by sandeep570** [reply]
I am using latest version of Dompdf and I am using Magento CE 2.4.1. Using composer I have installed but still facing error that class not found. Also I am using PHP 7.4 so why the issue of class not found is causing the issue?? Could you please help me out with this?

**Reply by bsweeney** [reply]
Are you referencing the composer autoloader? That would be my initial guess. Would also be helpful to see some sample PHP.

**Comment by sandeep570** [comment]
Hello Brian,

I am using latest version of Dompdf and I am using Magento CE 2.4.1. Using
composer I have installed but still facing error that class not found. Also
I am using PHP 7.4 so why the issue of class not found is causing the
issue?? Could you please help me out with this?

Thanks & Regards,
Sandeep.
